src/cursor/cur_file.c

```diff
@@ -719,9 +719,11 @@ __curfile_create(WT_SESSION_IMPL *session, WT_CURSOR *owner, const char *cfg[],
         cursor->modify = __curfile_modify;
 
     /*
-     * WiredTiger.wt should not be cached, doing so interferes with named checkpoints.
+     * WiredTiger.wt should not be cached, doing so interferes with named checkpoints. TODO:
+     * WT-5477: WiredTigerHS.wt should be cached.
      */
-    if (cacheable && strcmp(WT_METAFILE_URI, cursor->internal_uri) != 0)
+    if (cacheable && strcmp(WT_METAFILE_URI, cursor->internal_uri) != 0 &&
+      strcmp(WT_HS_URI, cursor->internal_uri) != 0)
```

Comment:
At the moment we're not caching?

Reply:
Yeah, we had some issue with caching, WT-5477 which duplicates to WT-5501 is going to resolve that.